test/integration/test_task_run_diagnostic_reports.py

```diff
@@ -1,19 +1,37 @@
+import os
+import pytest
+
+from YTClient.YTClient import YTClient
+from YTClient.YTDataClasses import Command
+
 from src.task_run_diagnostic_reports import run_diagnostic_reports
 from test.integration.fake_smtp_utils import FakeSmtpUtils, FakeEmailProperties
-import pytest
+
 
 smtp = FakeSmtpUtils()
 
+yt_token = os.environ["YOUTRACK_TOKEN"]
+yt = YTClient('https://mrc-ide.myjetbrains.com/youtrack/',
+              token=yt_token)
+test_touchstone = "touchstone-task-runner-test"
+
 
 @pytest.fixture(scope="module", autouse=True)
-def mod_header(request):
+def cleanup_emails():
     smtp.delete_all()
 
 
+@pytest.fixture(autouse=True)
+def cleanup_tickets():
+    issues = yt.get_issues("summary: {}".format(test_touchstone))
```

Comment:
Is this searching for issues to delete which have a summary containing the test touchstone? I thought it was the report version url which is being used for the summary?
Anyway, there do seem to be a lot of these tickets hanging around: https://mrc-ide.myjetbrains.com/youtrack/issues/VIMC?q=Check%20%26%20share